doc/workshops/financial.rst

```diff
@@ -2,4 +2,1030 @@
+   opt_weights_2 = optimal_weights(rets_2, SYMBOLS_2, weights_2, start_year, end_year)
+   
+   port_2_ow = pd.DataFrame.from_dict(opt_weights_2, orient='index')
+   port_2_ow.columns = SYMBOLS_2
+
+.. image:: images/workshop-1/portfolio-2-optimal-weights.png
+   :alt: Portfolio 2 optimal weights by year
+
+In addition, we can use these optimal weights to plot the expected and realized returns of this portfolio for each year.
+
+.. code-block:: python
+
+   port_2_exp_real = pd.DataFrame.from_dict(exp_real_rets(rets_2, opt_weights_2, SYMBOLS_2, start_year, end_year), orient='index')
+   port_2_exp_real.columns = ["expected", "realized"]
+
+   port_2_exp_real.plot(kind="bar", figsize=(16, 10),title="Expected vs. realized portfolio returns")
+
+.. image:: images/workshop-1/expected-vs-realized-portfolio-2-returns.png
+   :alt: Portfolio 2 expected and realized returns comparison
+
+Due to the high volatility of this portfolio, our model has not been able to adequately forecast expected returns in several of the years. Higher than expected returns would have been realized in 2013 and 2017, but in 2015 and 2018 using the model would have generated losses.
+
+Finally, let us look at the differences between the expected and realized means, and the linear correlation between the data.
+
+.. code-block:: python
+
+   print("Expected and realized means:")
+   print(port_2_exp_real.mean())
+   print("Expected and realized correlations:")
+   print(port_2_exp_real[["expected", "realized"]].corr())
+
+   # Expected and realized means:
+   # expected    0.155103
+   # realized    0.062207
+
+   # Expected and realized correlations:
+   #          expected  realized
+   # expected  1.000000 -0.023134
+   # realized -0.023134  1.000000
+
+As we can see, the model with optimal weights predicted a return close to 15% per year, but the realized return would have barely reached 6%. And the negative correlations show, as with portfolio 1, that there does not seem to be any correspondence between these values.
+
+The comparison is then favorable for portfolio 1. But what if we compare portfolio 1 with a "higher risk" investment such as cryptocurrencies? Let's discuss it below.
+
+
+
+===============
+Third portfolio
+===============
+
+Download cryptocurrencies data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our third portfolio will consist of three cryptocurrencies: bitcoin (BTC), ethereum (ETH) and litecoin (LTC). To access historical data, we are going to use a library called **Historic-Crypto**.
+
+.. important::
+
+   The Python Historic-Crypto library is not found in the Anaconda repository. It can be found in the PyPI repository and can be downloaded with the Pip package manager as follows: ``>>> pip install Historic-Crypto``. If you do not want to install this library, download the dataset "crypto_hist.csv" in your working directory, and load it in memory with the instruction ``crypto_hist = pd.read_csv("crypto_hist.csv")``, and skip the rest of this section.
```

Comment:
Does this mean we expect people to use pip from inside the ipython terminal? Just to confirm.

Reply:
This instruction was at the beginning of the workshop, why put it here?

Reply:
Fixed.